source/deployment/deployment.md

```diff
@@ -166,7 +188,7 @@ The [Mattermost Push Notification Service](http://docs.mattermost.com/deployment
 
 ### Mobile devices without VPN clients
 
-Mattermost can be deployed outside your private network by opening standard ports like 80 or 443. With this option, mobile clients and desktop computers access the Mattermost server without a VPN client, and it is recommended that users sign in with multi-factor authentication, available in Mattermost E10 and higher.
+If the business owner chooses to offer Mattermost via mobile applications available on the internet, by opening ports such as 80 and 443, Mattermost Enterprise Edition-specific two-factor authentication ("2FA") features via a single-sign-on provider supporting 2FA like Okta, OneLogin, SAML-based SSO or similar, or a setup with user login credentials plus Google Authenticator.
```

Comment:
Like above, the "If the business owner" and listing the ports seems unnecessary. This could start with "If Mattermost is available on the internet, ...", and similar comments about listing SSO providers and using 2FA vs MFA.

And I don't think this is actually a sentence since we just list login providers and MFA in the second part of the sentence without saying anything about them. It could be reworded "we recommend Mattermost Enterprise Edition featuring SAML-based single sign-on and multi-factor authentication (MFA) using Google Authenticator". Or if you want the more detailed version, "we recomment Mattermost Enterprise Edition featuring SAML-based single sign-on, such as Okta or OneLogin, and muti-factor authentication (MFA) using Google Authenticator"